### ISSUE TITLE
[6.0] Vendor migrations randomly prioritized over published ones

### ISSUE THREAD

**Issue by autaut03** [laravel/framework#29912]
- Laravel Version: 6.0.1
- PHP Version: 7.3.6
- Database Driver & Version: MariaDB

### Description:
After upgrading to 6.0 and re-migrating the database, some of my published (`vendor:publish`) `laravel/passport` migrations weren't being called. Instead, migrations from `/vendor/laravel/passport` were used, but not all of them. For example, published migration `CreateOauthAuthCodesTable` is always used over same migration from `vendor`, but `CreateOauthAccessTokensTable` is always used from `vendor`.

Obviously, this means I am not able to customize migrations, unless `/vendor/laravel/passport/database/migrations` folder is manually cleared.

I am unsure whether it's an issue of `laravel/passport`'s autoloading or the framework itself, so I'm posting the issue here.

### Steps To Reproduce:
1. Install Laravel 6
2. Install `laravel/passport`
3. Publish the migrations - `vendor:publish`
4. Customize published migrations
5. Migrate

You'll see that some of the published won't even be called.

**Comment by xuanquynh** [comment]
I can't reproduce your issue. Everything works properly. You may run `composer dump-autoload` and try `migrate` command again.

**Comment by driesvints** [comment]
Have you put the `Passport::ignoreMigrations` in your `register` method of your `AppServiceProvider`? More about that in the docs here: https://laravel.com/docs/6.0/passport#installation

**Comment by autaut03** [comment]
Thanks for quick responses.

`dumpautoload` was one of the first things I tried.

I have put `Passport::ignoreMigrations` in one of my providers, but not `AppServiceProvider` one. One it was in is being registered on the fly, using `$this->app->register($provider);`, but I made a mistake of registering other provider inside `boot` method, so even though `Passport::ignoreMigrations` call was in `register`, it was called too late.

Anyway, thanks both of you.
